content/docs/demos/outbound_ip_exclusion.md

```diff
@@ -35,7 +35,7 @@ The following demo shows an HTTP `curl` client making HTTP requests to the `http
     osm namespace add curl
 
     # Deploy curl client in the curl namespace
-    kubectl apply -f docs/example/manifests/samples/curl/curl.yaml -n curl
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/samples/curl/curl.yaml -n curl
```

Comment:
Should this url point to the yaml file on a release branch instead of on main?

Reply:
Based on #191 we decided that docs on the main branch here link to the main branch of the osm repo.